model/Core/Classes/Location.md

```diff
@@ -0,0 +1,31 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# Location
+
+## Summary
+
+Location is used to define the location, address or GPS coordinates of an object.
+
+## Description
+
+Mailing, street and description define location. GPS coordinates are used to locate any object anywhere on earth.
+
+## Metadata
+
+- name: Location
+- Instantiability: Concrete
+
+## Properties
+- locationTime
+  - type: DateTime
+  - minCount: 1
+  - maxCount: 1
+- mailingAddress
```

Comment:
Consider just having "Address", and then provide type (mailing, street, geographic, electronic)

Reply:
https://schema.org/location has four types:

- Place
  - can contain [Global Location Number (GLN)](https://www.gs1.org/standards/id-keys/gln)
- PostalAddress
- Text
- VirtualLocation